OWNERS_ALIASES

```diff
@@ -32,6 +32,10 @@ aliases:
     - soltysh
     - pwittrock
     - AdoHe
+  sig-cloud-provider-leads:
+    - andrewsykim
+    - jagosan
```

Comment:
cc @jagosan @wlan0

Reply:
As the author of the original charter, someone who's worked closely with the sig-docs and sig-testing teams, and has been an active participant in the group that has openly advocated to the benefit of all cloud providers, I would also like to be named as a co-lead in this sig.

Reply:
+1 @hogepodge

Reply:
Any concerns with deadlocks if the number of leads is even?  How are decisions arbitrated?

Reply:
Spoke with Jago and Sid, Sid (wlan0) plans to step down as chair (due to other obligations) so we should be good to have Chris onboard while also addressing @jdumars concerns around deadlocks.